docs/spark/how-to-guides/dotnet-spark-jupyter-notebooks.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Use Jupyter notebooks
+titleSuffix: .NET for Apache Spark
+description: Use .NET for Apache Spark in interactive environments like Jupyter Notebook, Jupyter Lab, or Visual Studio Code (VS Code)
+ms.author: luquinta
+author: luisquintanilla
+ms.date: 10/09/2020
+ms.topic: conceptual
+ms.custom: mvc, how-to
+---
+
+# Use .NET for Apache Spark in Jupyter notebooks
+
+In this article, you learn how to run .NET for Apache Spark jobs interactively in Jupyter Notebook and Visual Studio Code (VS Code) with .NET Interactive.
+
+## About Jupyter
+
+[Jupyter](https://jupyter.org/) is an open-source, cross-platform computing environment that provides a way for users to prototype and develop applications interactively. You can interact with Jupyter through a wide variety of interfaces such as Jupyter Notebook, Jupyter Lab, and VS Code.
+
+In the context of .NET, [.NET Interactive](https://github.com/dotnet/interactive), a .NET Core global tool, provides a kernel for writing .NET code (C#/F#) in interactive computing environments such as Jupyter Notebook.
+
+## Prerequisites
+
+- [.NET Core 3.1 SDK](https://docs.microsoft.com/dotnet/core/install/)
+- Apache Spark
```

Comment:
Can we add a link to Apache Spark official website or something here?

Reply:
done